src/src/common/middlewares/authmiddleware.ts

```diff
@@ -9,6 +10,10 @@ export class AuthMiddleware implements NestMiddleware {
 		if (req.headers.authorization) {
 			const user = await this.userService.ipUserInfo(req);
 			req.mw_userid = user?.data?.id;
+			if (user) {
+				const decoded: any = jwt_decode(req.headers.authorization);
+				req.mw_roles = decoded.resource_access.hasura.roles;
```

Comment:
set default value for req.mw_roles as [] (blank array / list)

Reply:
Done